### DISCUSSION TITLE
2.0.0-alpha.14 vs 1.9.1?

### DISCUSSION THREAD

**Discussion by chrislgrigg** [dinerojs/dinero.js#746] [Q&A] [answered]
Any existing users able to comment on which one to use? Is the alpha significantly better than the last stable release, so much so that it's worth using despite being alpha? Or is the 1.9.1 release still viable for production usage?

**Comment by diegoazh** [comment]
I have the same question, can I use the v2 in production safety or should I use the stable v1?

**Comment by sarahdayan** [comment, marked as answer]
Hey @subvertallchris, @diegoazh! The v2 alpha is production-ready and I'd recommend it for new projects. The API is stable and well tested. The "alpha" label reflects that a few things are still being finalized before a stable release, but there shouldn't be any major breaking changes at this point.

v1 is no longer maintained, so if you're starting fresh, go with v2.
